index.bs

```diff
@@ -1955,7 +1955,7 @@ Consensus</h4>
 	or <a href="#distributed-meeting">distributed</a>)
 	as well as through <a href=#discussion-archiving>persistent text-based discussions</a>.
 
-	Note: The Director, CEO, and COO have the role of
+	Note: The Director and CEO have the role of
 	assessing consensus within the Advisory Committee.
```

Comment:
As long as this is being massaged, `have the role of assessing consensus` should become either `have the role of consensus assessor` or `perform the task of assessing consensus` ... or something requiring a bit more bikeshedding.

Reply:
I have objection to further improving this Note, however this edit seems orthogonal to removal of "COO", so would prefer it be done in another PR, especially since it appears to request some discussion of options.